### PR TITLE
NO-JIRA: Additional exceptions for ROSA CI

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: pkg/monitortests/node/legacynodemonitortests/exclusions.go [hunk 1]
```diff
@@ -47,6 +47,13 @@ func isThisContainerRestartExcluded(locator string, exclusion Exclusion) bool {
 		{
 			containerName: "container/ovn-acl-logging", // https://issues.redhat.com/browse/OCPBUGS-42344
 		},
+		{
+			containerName: "container/managed-upgrade-operator", // https://issues.redhat.com/browse/OSD-26270
```
Comment (deads2k) [inline pkg/monitortests/node/legacynodemonitortests/exclusions.go:51]: Oof. How?  Won't block merge, but I really hope this stabilizes

File: pkg/monitortests/node/legacynodemonitortests/exclusions.go [hunk 2]
```diff
@@ -47,6 +47,13 @@ func isThisContainerRestartExcluded(locator string, exclusion Exclusion) bool {
...
+		},
+		{
+			// Managed services like ROSA. This is expected.
+			containerName: "container/osd-cluster-ready",
```
Comment (deads2k) [inline pkg/monitortests/node/legacynodemonitortests/exclusions.go:55]: Irony. Also non-blocking

File: pkg/monitortests/node/legacynodemonitortests/exclusions.go (hunk 1 above)
Comment (stbenjam) [reply pkg/monitortests/node/legacynodemonitortests/exclusions.go:51]: Yea I'm chasing them for fixes for all these, but trying to get the job to pass so I can try to hold on to what I've got working

**Pull request merged**